### DISCUSSION TITLE
PorousFlow doc on GitHub

### DISCUSSION THREAD

**Discussion by sinaomrani96** [idaholab/moose#23412] [Q&A Modules: General] [answered]
Hello,
Hope you are doing well.

Just wanted to inform you that PorousFlow doc on Github faces "404 not found error" or "Unable to render expression". Nothing important, just wanted to let you know if you want to fix it. Thanks.

Like theses:
https://github.com/idaholab/moose/blob/next/modules/porous_flow/doc/content/modules/porous_flow/capillary_pressure.md#constant
https://github.com/idaholab/moose/blob/next/PorousFlowCapillaryPressureConst.md

Best regards,
Sina

**Comment by sinaomrani96** [comment]
Well it is not just PorousFlow. Lots of hyperlinks do not work in doc directories.

https://github.com/idaholab/moose/blob/next/UserObjects/PropertyReadFile/nprop


Best regards,
Sina

**Comment by GiudGiud** [comment, marked as answer]
Hello

You are accessing these links from the github interface, but they are not made for this. They need to be parsed by our documentation system to create the website. 

Please see the following pages that correspond to the links you pasted:
https://mooseframework.inl.gov/modules/porous_flow/capillary_pressure.html
https://mooseframework.inl.gov/source/userobjects/PorousFlowCapillaryPressureConst.html
https://mooseframework.inl.gov/source/userobject/PropertyReadFile.html

Best,
Guillaume
